scss/_themes.scss

```diff
@@ -3,8 +3,11 @@
 /// @link http://registry.origami.ft.com/components/o-buttons
 ////
 
-// Set custom color white over black @ 20%
-@include oColorsSetColor('white-black-20', oColorsMix($color: 'white', $background: 'black', $percentage: 20));
+// Set custom color white over black @ 30%
+@include oColorsSetColor('white-black-30', oColorsMix($color: 'white', $background: 'black', $percentage: 30));
```

Comment:
The bug here is not the 20% needing to be 30%, it's that the colour mixes `white` and `black` when it should be `white` and `slate`